rfc6962-bis.xml

```diff
@@ -1643,13 +1643,7 @@ corrective action when a misissue is detected.
       </section>
       <section title="Redaction of Public Domain Name Labels">
         <t>
-          CAs SHOULD NOT redact domain name labels in precertificates such that the entirety of the domain space below the unredacted part of the domain name is not owned or controlled by a single entity
-(e.g. <spanx style="verb">?.com</spanx> and
-<spanx style="verb">?.co.uk</spanx> would both be problematic). Logs
-MUST NOT reject any precertificate that is overly redacted but which is
-otherwise considered compliant. It is expected that monitors will treat overly
-redacted precertificates as potentially misissued. TLS clients MAY reject a
-certificate whose corresponding precertificate would be overly redacted, perhaps using the same mechanism for determining whether a wildcard in a domain name of a certificate is too broad.
+          Redaction of domain name labels carries the same risks as the use of wildcards (See Section 7.2 of <xref target="RFC6125"/>, for example). CAs should take care not to redact domain name labels in precertificates too broadly, so that the entirety of the domain space below the unredacted part of the domain name may be controlled by multiple entities, such as <xref target="Public.Suffix.List">public suffixes</xref> (e.g. <spanx style="verb">?.com</spanx> and <spanx style="verb">?.co.uk</spanx> would both be problematic). Logs MUST NOT reject any precertificate that is overly redacted but which is otherwise considered compliant. It is expected that monitors will treat overly redacted precertificates as potentially misissued. TLS clients MAY reject a certificate whose corresponding precertificate would be overly redacted, perhaps using the same mechanism for determining whether a wildcard in a domain name of a certificate is too broad.
```

Comment:
Adopted your suggestion, with the difference of saying TLS clients MAY consider a certificate non-compliant if it's overly redacted. (they may do other things, of course, like rejecting it)